mundane-bytes/src/test/scala/com/ambiata/mundane/bytes/BufferSpec.scala

```diff
@@ -0,0 +1,47 @@
+package com.ambiata.mundane.bytes
+
+import com.ambiata.disorder._
+import org.specs2.{ScalaCheck, Specification}
+
+class BufferSpec extends Specification with ScalaCheck { def is = s2"""
+
+  $wrapArray
+  $wrapArrayAndEmpty
+  $allocateNoResize
+  $allocateResize
+  $allocateHasCapacity
+  $allocateCopy
+"""
+
+  // TODO Add offset change and then allocate!
```

Comment:
> I think if you add a way to construct a "populated" array

Just to confirm - you just mean an `Arbitrary[Buffer]`? Will do first thing in the morning.